### PR TITLE
Backend and frontend improvements

### PR DESCRIPTION
**Pull request by lakshyakalia**
- Alternate phone number is now also being sent to frontend
- Extra variables which were declared in order component are completely removed and in place of those only an object is used in html to show the values

### CODE REVIEW

File: frontend/uv-poc/src/app/order/order.component.ts [hunk 1]
```diff
@@ -35,24 +24,12 @@ export class OrderComponent implements OnInit {
 
     this.orderDetailService.get(this.transactionId).subscribe(
       (res: any) => {
-        this.setValues(res['orderDetail']);
+        this.orderData = res['orderDetail'];
+        console.log(this.orderData);
```
Comment (KartikyaThakur) [inline frontend/uv-poc/src/app/order/order.component.ts:28]: No `console.log` statements, please

Comment (KartikyaThakur) [inline frontend/uv-poc/src/app/order/order.component.ts]: No `console.log` statements, please

**Review by KartikyaThakur — CHANGES_REQUESTED** [review]

File: frontend/uv-poc/src/app/order/order.component.ts (hunk 1 above)
Comment (lakshyakalia) [reply frontend/uv-poc/src/app/order/order.component.ts]: sorry @KartikyaThakur

Comment (KartikyaThakur) [reply frontend/uv-poc/src/app/order/order.component.ts]: Don't be sorry, just remove the `console.log` statement

**Review by KartikyaThakur — CHANGES_REQUESTED** [review]

**Pull request merged**